packages/x-components/src/components/icons/tag.vue

```diff
@@ -0,0 +1,20 @@
+<template functional>
+  <svg
+    :class="['x-icon'].concat(data.staticClass, data.class)"
+    viewBox="0 0 24 24"
+    version="1.1"
+    xmlns="http://www.w3.org/2000/svg"
+    xmlns:xlink="http://www.w3.org/1999/xlink"
+  >
+    <g fill="none" fill-rule="evenodd" stroke-linecap="round" stroke-linejoin="round">
+      <!-- eslint-disable max-len -->
+      <path
+        d="m6.336 13.594 4.143 4.144a1.155 1.155 0 0 0 1.635 0l4.963-4.959V7h-5.778l-4.963 4.964a1.156 1.156 0 0 0 0 1.63zM14.188 9.89h-.006"
+      />
+    </g>
+  </svg>
```

Comment:
I think you can clean it up a little more:

```suggestion
  <svg :class="['x-icon'].concat(data.staticClass, data.class)" viewBox="0 0 24 24">
    <!-- eslint-disable max-len -->
    <path
      d="m6.336 13.594 4.143 4.144a1.155 1.155 0 0 0 1.635 0l4.963-4.959V7h-5.778l-4.963 4.964a1.156 1.156 0 0 0 0 1.63zM14.188 9.89h-.006"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
  </svg>
```

Reply:
For what I've read, these attributes are unnecessary if the SVG is going to be inlined inside HTML https://developer.mozilla.org/en-US/docs/Web/SVG/Element/svg.